linux_os/guide/system/software/sudo/sudo_add_env_reset/rule.yml

```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+prodtype: rhel7,rhel8
+
+title: 'Ensure sudo Runs In A Minimal Environment - sudo env_reset'
+
+description: |-
+    The sudo <tt>env_reset</tt> tag, when specified, will run the command in a minimal environment,
+    containing the TERM, PATH, HOME, MAIL, SHELL, LOGNAME, USER and SUDO_* variables.
+    On {{{ full_name }}}, <tt>env_reset</tt> is enabled by default
```

Comment:
I am not sure that we can guarantee this across everything. Maybe drop the line?

Reply:
Yeah, that's why I restricted the rule with a `prodtype`.
Maybe this line should be there only for `rhel7` and `rhel8`?

Reply:
I see this as fragile, because somebody may update the prodtype without examining the description. Moreover, the rest of rule works with every product that ships `sudo`. So I would incline towards a Jinja construct "if prodtype in ..., then ...".

Reply:
Okay, I have restricted the description line to `rhel7` and `rhel8` along with the template `defualt_is_enable`.